include/dump_utils.hpp

```diff
@@ -21,10 +21,11 @@ namespace dump_utils
 inline void getValidDumpEntryForAttachment(
     const std::shared_ptr<bmcweb::AsyncResp>& asyncResp, const std::string& url,
     std::function<void(const std::string& objectPath,
-                       const std::string& entryID,
-                       const std::string& dumpType)>&& callback)
+                       const std::string& /*entryID*/,
+                       const std::string& /*dumpType*/)>&& callback)
```

Comment:
why to comment out these args?

Reply:
> why to comment out these args?

They are just prototype of the function.  In most cases, they are even defined.
Another reason is because those names are also used in the function body and thus I'd like to avoid the confusion.